CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
     - `-auth-logging-format` Sets the format for auth logging
 
 - [#111](https://github.com/pusher/oauth2_proxy/pull/111) Add option for telling where to find a login.gov JWT key file (@timothy-spencer)
+- [#170](https://github.com/pusher/oauth2_proxy/pull/170) Restore tarball contents to be compatible with bitlys original tarballs (@zeha)
```

Comment:
I think this would count as a breaking change, so may be worth adding a quick note to the breaking changes section above this

Reply:
Added a (more detailed) note to breaking changes.